diff --git a/src/google/protobuf/stubs/common.h b/src/google/protobuf/stubs/common.h
index 8b8cbc1..05d6ebc 100644
--- a/src/google/protobuf/stubs/common.h
+++ b/src/google/protobuf/stubs/common.h
@@ -177,7 +177,7 @@ class FatalException : public std::exception {
       : filename_(filename), line_(line), message_(message) {}
   virtual ~FatalException() throw();
 
-  virtual const char* what() const throw();
+  const char* what() const noexcept override;
 
   const char* filename() const { return filename_; }
   int line() const { return line_; }
